integration_tests/components/prisonerSplitView.ts

```diff
@@ -0,0 +1,19 @@
+import { PageElement } from '../pages/page'
+
+export default class PrisonerSplitViewComponent {
```

Comment:
Added the concept of components into the integration tests. The reasoning is I think we could build up pages through the composition of components rather than defining each attribute to lessen some duplication and add some standardisation. Thoughts?